### ISSUE TITLE
Support `maxAge` query parameter to Shields.io in badge redirect

### ISSUE THREAD

**Issue by wmayner** [rtfd/readthedocs.org#3000]
## Details

Shields.io allows passing a `maxAge` query parameter that should address #1612, but this is not supported by the RTD redirect.

## Expected Result

The `Location` header from `curl -v https://readthedocs.org/projects/pyphi/badge/?version=latest&style=flat-square&maxAge=600` should be:
```
Location: https://img.shields.io/badge/docs-latest-brightgreen.svg?style=flat-square&maxAge=600
```

## Actual Result

`maxAge` is ignored, though `style` is respected.

The `Location` header from `curl -v https://readthedocs.org/projects/pyphi/badge/?version=latest&style=flat-square&maxAge=600` is:
```
Location: https://img.shields.io/badge/docs-latest-brightgreen.svg?style=flat-square
```

**Comment by ericholscher** [comment]
This would be a great contribution! The code is here: https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/projects/views/public.py#L119

**Comment by wmayner** [comment]
Implemented in PR #3005.
